models/user.go

```diff
@@ -712,18 +712,50 @@ func (u *User) GetOwnedOrganizations() (err error) {
 
 // GetOrganizations returns paginated organizations that user belongs to.
 func (u *User) GetOrganizations(opts *SearchOrganizationsOptions) error {
-	ous, err := GetOrgUsersByUserID(u.ID, opts)
+	sess := x.NewSession()
+	defer sess.Close()
+
+	schema, err := x.TableInfo(new(User))
 	if err != nil {
 		return err
 	}
+	groupByCols := &strings.Builder{}
+	for _, col := range schema.Columns() {
+		fmt.Fprintf(groupByCols, "`%s`.%s,", schema.Name, col.Name)
```

Comment:
do you forgot to remove debugging code?

Reply:
This generates group by string

Reply:
ah I got it, this way you can avoid the x.IN(select query).Get/Del construct ...